### PR TITLE
Add Fear of Change

### PR DESCRIPTION
**Pull request by TwentyToedToad**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/fear_of_change.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Fear of Change
+ManaCost:G U
+Types:Enchantment Creature Nightmare
+PT:2/3
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Battlefield | OptionalDecider$ You | Execute$ TrigExile | TriggerDescription$ When this creatures enters or dies, exile another creature you control. If you do, conjure a duplicate of a random creature card with mana value X onto the battlefield, where X is 2 plus the exiled creature's mana value.
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Graveyard | OptionalDecider$ You | Execute$ TrigExile | Secondary$ True | TriggerDescription$ When this creatures enters or dies, exile another creature you control. If you do, conjure a duplicate of a random creature card with mana value X onto the battlefield, where X is 2 plus the exiled creature's mana value.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fear_of_change.txt:6]: not optional

File: forge-gui/res/cardsfolder/upcoming/fear_of_change.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+SVar:TrigExile:DB$ ChangeZone | ValidTgts$ Creature.Other+YouCtrl | TgtPrompt$ Exile another creature you control. | Origin$ Battlefield | Destination$ Exile | SubAbility$ DBNameCard
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/fear_of_change.txt:7]: doesn't target

Comment (TwentyToedToad) [reply forge-gui/res/cardsfolder/upcoming/fear_of_change.txt]: Updated! Took inspiration from [Yannik, Scavenging Sentinel](https://scryfall.com/search?as=grid&order=name&q=oracle%3A%22exile+another+creature+you+control%22+%28game%3Apaper%29)

**Review by tool4ever — APPROVED** [review]
the extra ChooseCard isn't really necessary but good enough for Alchemy

**Pull request merged**